APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+* What monetary value one puts on experience (e.g. an undergraduate student vs. an experienced software developer).
+
+`NEP 48 <https://numpy.org/neps/nep-0048-spending-project-funds.html>`_ has extensive discussion and a good list of further references on this topic.
+
+We recognize though that the approach of leaving the budget entirely up to project members also has downsides: In particular, less experienced people may be tempted to suggest work packages with a compensation that is unfairly low for the work required to increase their perceived chances to obtain funding. We place trust in the community to recognize these short-comings of the process and to advise and mentor each other to propose realistic budgets that allow the proposed work to be completed with a fair rate of pay. As the diversity of proposers increases, we recognize that this approach will likely need to be modified and potentially be made more prescriptive. This may come in the form of a future APE or more incremental changes as experience grows.
+
+Defining fundable activities and projects
+-----------------------------------------
+We define three separate categories of expense, by value:
+
+* Up to $2000 (US);
+* $2000 to $10000 (US);
+* More than $10000 (US).
+
+Items in the first category — up to $2000 — may be approved by majority vote of the Finance Committee.
+
+Items in the second category — up to $10000 — can be approved by majority vote of the Finance Committee. If, in the judgment of one or more Finance Committee members, the item would not receive widespread support, the Committee will communicate these concerns to the CoCo. The CoCo may provide a decision, or may request that the Finance Committee organize a vote among the Astropy membership to approve the expense.
+
+Items in the third category — more than $10000 — should follow approved Project decision making methods from APE0, or a vote among the Astropy voting membership unless specifically allocated in a grant to a specific person or organization already; in the latter case the Finance Committee is tasked to ensure it is spend according to the provisions of the grant. The Finance Committee will periodically solicit suggestions from the community for projects and other major expenses and coordinate a selection process.
+
+*Rationale: We want the community to make the big and important decisions, but also not overburden community members with frequent discussions and voting.*
+
+Strategy for spending/saving funds
+----------------------------------
+There is an expectation from Astropy funders that the funds will be used for the benefit of the project and the community. Therefore, we should spend available funds thoughtfully, strategically, and fairly, as they become available. For emergencies, we should keep a reserve of $10,000 to $15,000 (US) which could cover, for example, a year of CI and hosting services, 1–2 months of full-time maintenance work, or engaging a consultant to address a specific need.
```

Comment:
At $150/h (seems typical from previous proposals), 7h/day, 5 days/week, 4 weeks/month, this figure seems to be ~2 times lower than what is needed to cover 1-2 months of full time maintenance work.

Reply:
Would you suggest to increase it? Not everybody is paid as independent contractor (some people get sub-grants), we don't know if we keep that pay scale for everybody in the future (discussion on paying fairly is above), e.g. a undergrad student might ask for less than an experienced professional software developers. Maybe we should jsut say "a few weeks"?

Reply:
I guess that depends critically on what the emergency is - and we don't know that in advance. I'll go with "a few weeks" which means "not nothing, also also don't stash the money" leaving the definition of how many weeks exactly in the hands of the finance committee planning. In practice, that's not a problem right now since we have a number of contributors that are either paid by their institutions or through currently running contacts - either of them would likely step in in the case of that emergency. So, so only real data point we have is that sudden move from Travis to GH action. While I don't hope we get more emergencies, it's hard to put a number that's based on experience here at this point.

Reply:
If there are emergencies that you think might happen, the scenario should be defined and costed. For example one scenario might be "All CI stops being free". You can cost that up, and add it to the total reserves required in case that happens. Has anyone actually defined those scenarios, and done those (back of the envelope) calculations to work out what a reasonable amount of reserves is? Why $10,000 and not $1000 or $100,000?

Reply:
The sum is taken form Numpy's NEP 48, but for astropy we have not defined a range of scenarios that should be costed for this case. To a certain degree, this is more pragmatic than that: We know that 1000 $ does not get us far, and 1e5 $ is more than we have. As an order of magnitude estimate, we can look at the Travis - > github actions switch. That took of order "a few weeks" of work (distributed over a number of people). But even that's not a hard number. In an emergency, is it sufficient to fix astropy core and infrastructure packages? How about coordinated packages? How much CI do we really need? We could probably reduce the number of runs without missing too many errors if we really have to. 

So, a second scenarios would be "Github does not offer free tier and longer". In the long run that would probably mean that we'll move away form github, but as a point of costing, let's assume that we'll pay for the cheapest paid plan they currently have. That's 48 $ / user / year. We probably have 100-200 contributors that would need access, so that's about 5-10 k$.